libs/blocks/global-navigation/global-navigation.js

```diff
@@ -182,6 +184,7 @@ class Gnav {
       loadBaseStyles,
       this.decorateMainNav,
       this.decorateTopNav,
+      this.decorateAside,
       this.decorateTopnavWrapper,
       this.ims,
       this.addChangeEventListeners,
```

Comment:
We should create a Ticket to investigate a graceful failover strategy if any of those tasks fail - the nav will be gone

Reply:
Well, they rely on each other anyway, so essentially if anything before `this.decorateTopnavWrapper` fails we wouldn't show a navigation. It wouldn't make sense to just show some bits and pieces that actually succeeded, like a nav with just Search and Breadcrumbs, either. IMS has its own catch, while `this.addChangeEventListeners` has all the safety checks in place. Lastly, the whole `init` method is wrapped in a `logErrorFor` call, so I think there should be enough safety mechanisms in place. If we start seeing `"Error in global navigation init"` in our logs, we can look into this more.